components/elm/cime_config/config_pes.xml

```diff
@@ -523,4 +523,42 @@
       </pes>
     </mach>
   </grid>
+  <grid name="a%ne30np4.pg2_l%r05_oi%null_r%r05_g%mpas.gis20km_w%null_z%null_m%EC30to60E2r2">
+    <mach name="chrysalis|pm-cpu|alvarez">
+      <pes compset="any" pesize="any">
+        <comment>GIS 20km (low-res) testing config</comment>
+        <MAX_MPITASKS_PER_NODE>128</MAX_MPITASKS_PER_NODE>
+        <MAX_TASKS_PER_NODE>128</MAX_TASKS_PER_NODE>
+        <ntasks>
+          <ntasks_atm>128</ntasks_atm>
+          <ntasks_lnd>128</ntasks_lnd>
+          <ntasks_rof>128</ntasks_rof>
+          <ntasks_ice>128</ntasks_ice>
+          <ntasks_ocn>128</ntasks_ocn>
+          <ntasks_glc>128</ntasks_glc>
```

Comment:
It might be worth retrying this with https://github.com/stephenprice/E3SM/pull/1 . The current albany install was using a higher SFAD size which increases memory usage (possibly by exactly 4x).

Reply:
Thanks @jewatkins. I'm not sure what I need to do to allow you to add this commit to my branch. Let me know if you do. I'm happy to add it and test it out.

Reply:
@stephenprice If you're okay with the changes, there should be a "Merge" button at the bottom of this page https://github.com/stephenprice/E3SM/pull/1. That should add my changes.

Reply:
@trhille -- right now, there's only a default PE layout for the IG case for the 20km mesh. I will add one now for IG cases using the 1-to-10km grid. Thanks for catching that.

Reply:
I've added a default for this that works (tested on Chyrs). I will push that momentarily so I'm going to resolve this for now.